.buildkite/pipeline.yml

```diff
@@ -56,6 +56,11 @@ steps:
       ./result/bin/baseDAO-test --nettest-run-network
       -A localhost -P 8735
 
+  - label: ligo
+    if: *not_scheduled
+    commands:
+    - nix-build ci.nix -A build-ligo
```

Comment:
I think we should also put the resulting contract into artifacts.

Reply:
I tried hard but failed, will offload this to Ops team (hopefully after this PR is merged, because I don't want to block anyone).

Reply:
I haven't added CI test execution either because of this (they need the resulting contract), I also spent some time on it without luck and prefer to do so in a separate PR so that this one can be merged.